koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -619,6 +624,16 @@ def handler(self, src, target, opts=None):
                                       "target": target}
             if not SCM.is_scm_url(src):
                 raise koji.BuildError('Invalid source specification: %s' % src)
+            # Scratch builds shouldn't be checked for nvr
+            if not self.opts.get('scratch'):
+                expected_nvr = labels_wrapper.get_expected_nvr()
+                try:
+                    build_id = self.xmlrpc.getBuild(expected_nvr)['id']
+                except:
+                    self.logger.info("No build for %s found" % expected_nvr)
```

Comment:
Could change "%" to "," to use lazy string formatting provided by logging functions.